doc/content/lorawan/db-dbm-dbi-dbd/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "dB, dBm, dBi, and dBd"
+section: The Things Advanced
+description: ""
+weight:
+---
+
+In this chapter we learn about some units in brief that are used to measure the performance of transceivers (gateways and end devices) and antennas.
+
+## dB (decibel)
+
+Decibel expresses the ratio between two power levels, for example, the output power and the input power of an amplifier. The input power (P<sub>in</sub>) and output power (P<sub>out</sub>) are usually measured in Watt (W) or milliwatt (mW).
+
+For example, an amplifier with an output of 1000 Watt when the input is 1 Watt, its power ratio or amplification factor can be calculated as,
+
+**P<sub>out</sub>/P<sub>in</sub>** = 1000/1 = 1000
+
+Where P<sub>out</sub> is the output power and the P<sub>in</sub> is the input power. Furthermore, we can modify the above formula to get the answer on a logarithmic scale, in this case we use log<sub>10</sub>.
+
+**dB = 10 log<sub>10</sub> P<sub>out</sub>/P<sub>in</sub>** = 10 log<sub>10</sub> 1000/1 = 30 dB
+
+If you look at the above two answers, you realize that the logarithmic scale helps us to avoid very large numbers (also avoids very small numbers). The table below shows how the power ratio is related to decibel. Also note that the number 0s in the power ratio corresponds to the dB value, for example, the 4 in 40 dB corresponds to the number of zeros in the power ratio.
+
```

Comment:
Somewhere around here we should also mention that 3dB means a doubling in power, and 6dB means quadrupling, as these are more useful shorthands to remember than 10x, 100x, etc.

Reply:
> Somewhere around here we should also mention that 3dB means a doubling in power, and 6dB means quadrupling, as these are more useful shorthands to remember than 10x, 100x, etc.

Will do